src/kubernetes-tool/utils/flags.ts

```diff
@@ -0,0 +1,4 @@
+export const flags: string[] = []
+
+// @ts-ignore
```

Comment:
Only ignore the specific rule that's failing, so we can still lint as much as possible?